packages/framework/src/Command/ChangeAdminPasswordCommand.php

```diff
@@ -57,6 +57,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->administratorFacade->changePassword($adminUsername, $password);
 
         $output->writeln(sprintf('Password for administrator "%s" was successfully changed', $adminUsername));
+
+        return 0;
```

Comment:
I would prefer this to be constant

Reply:
Should I create in all commands constant with zero value and return it? Or should I create one constatn somewhere (where?).

Reply:
Well best solution would be to have some ShopsysCommand extending Command so we could do some more thing in future, but I will be ok with any solution you will provide :)

Reply:
917e93545b41c043938afe2487f534d0ed8fa5d6